docs/spelling_wordlist.txt

```diff
@@ -0,0 +1,51 @@
+AWS
```

Comment:
could we add AMI too?

Reply:
I think I can drop AWS since capitalized words are skipped.